index.js

```diff
@@ -143,30 +143,31 @@ module.exports = {
       })
     }
 
-    function secretKeyFromPurpose(purpose) {
-      return new SecretKey(Buffer.from(purpose, 'base64'))
+    function secretKeyFromString(string) {
+      return new SecretKey(Buffer.from(string, 'base64'))
     }
 
     function createGroupWithoutMembers(myRoot, cb) {
-      const content = {
-        type: 'group/init',
-        tangles: {
-          group: { root: null, previous: null },
-        },
-      }
-      if (!initSpec(content)) return cb(new Error(initSpec.errorsString))
-
       findOrCreateGroupFeed(null, function gotGroupFeed(err, groupFeed) {
         // prettier-ignore
         if (err) return cb(clarify(err, 'Failed to find or create group feed when creating a group'))
 
-        const secret = secretKeyFromPurpose(groupFeed.purpose)
+        const secret = secretKeyFromString(groupFeed.purpose)
```

Comment:
Not a big deal, but a small suggestion:

```suggestion
        const groupKey = secretKeyFromString(groupFeed.purpose)
```

We seem to be going back and forth with this naming, "secret" or "group key", but we could try to be at least internally consistent (all namings of this thing in ssb-tribes2 are the same).

Reply:
Problem here is the spec calls it groupKey and the tribes2 api says secret. I think the proper way for fixing is a separate issue/pr.

Reply:
this pr is just for conforming to the spec

Reply:
Okay!